### PR TITLE
cukes being eliminated and replaced with rspec tests 

### PR DESCRIPTION
**Pull request by dostapenko**

### CODE REVIEW

File: spec/server_poll_checker_spec.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+require 'rspec'
+require 'pony'
+require 'pony'
```
Comment (oldfartdeveloper) [inline spec/server_poll_checker_spec.rb]: duplicate

File: spec/server_spec.rb [hunk 2]
```diff
@@ -22,12 +23,17 @@
     computer = DevelopmentComputer.new
     computer.receive_ipn(ipn_retrieved)
     computer.ipn.should == ipn
-
   end
 
-  #this test was erased because it is wrong. It is testing what happens when an (unknown or known) sandbox sends an IPN to the server
-  #when it is non-testing. It should just be bounced and no email should be sent
```
Comment (oldfartdeveloper) [inline spec/server_spec.rb:29]: 'bounced' meaning accepted/accepted or rejected?

Comment (dostapenko) [inline spec/server_spec.rb:29]: it means that the handshake is done with paypal and thats it. Nothing is stored. Not sure what you meant by "accepted/accepted or rejected"

File: spec/server_spec.rb [hunk 3]
```diff
@@ -94,25 +106,33 @@
...
+    it 'should sends email to all developers if no email on file' do
       Pony.should_receive(:mail).with(any_args).twice
-      @server.unexpected_poll('my_sandbox_unknown')
+      @server.respond_to_computer_poll('my_sandbox_unknown')
     end
 
-    it 'sends another notification if issue not handled 24 hours after previous email'
+    it 'should send another notification email if last email sent 24 ago as issue still not resolved' do
+      Pony.should_receive(:mail).with(any_args).twice
+      time = Time.now - 12*60*60
```
Comment (oldfartdeveloper) [inline spec/server_spec.rb]: Ruby/Rails will let you do

``` ruby
time = Time.now - 12.hours
```

(this is why developers love Ruby)

File: spec/server_spec.rb [hunk 4]
```diff
@@ -94,25 +106,33 @@
...
+      @server.respond_to_computer_poll('my_sandbox_unknown', time)
+      time_new = Time.now + 12*60*60
```
Comment (oldfartdeveloper) [inline spec/server_spec.rb]: same here

File: config_test.yml [hunk 5]
```diff
@@ -8,6 +8,8 @@ development_computer_url : 'http://localhost:3000/payments/ipn'
 
 polling_interval_seconds : '5.0'
 
+poll_checking_interval_seconds : '3600.0'
```
Comment (oldfartdeveloper) [inline config_test.yml:11]:
``` json
poll_checking_interval : '1.hour'
```

(probably not worth it)

File: spec/server_spec.rb (hunk 3 above)
Comment (dostapenko) [inline spec/server_spec.rb]: turns out that active_support gem is needed to make .hours work. We dont have the gem so Im going back to the previous way that this was done. i.e. 12_60_60

**Pull request merged**